sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -1394,6 +1394,7 @@ class DataSourceV2SQLSuite
 
   test("Use: v2 catalog is used and namespace does not exist") {
```

Comment:
we should update this test with catalogs that don't implement `SupportsNamespace`